haskell-org.cabal

```diff
@@ -6,7 +6,7 @@ cabal-version:      >= 1.10
 executable site
   main-is:          site.hs
   build-depends:    base == 4.*
-                  , hakyll == 4.12.*
+                  , hakyll == 4.13.*
```

Comment:
Could we loosen this to `hakyll >= 4.12 && < 4.13` ? Our code should still work on the old hakyll so I'd rather we not force a hakyll/ghc update on the build server.

Reply:
sure, let me give that a try